app/lib/presenter/comment.rb

```diff
@@ -18,5 +19,9 @@ def optics_payload
     def type
       ENV['COMMENT_TYPE'] || ''
     end
+
+    def submission_answers
```

Comment:
Do we have this already in the `BasePresenter` class? I can't remember if we ended up merging the `BasePresenter` class in tbh

Reply:
good spot! have updated